efficientdet/keras/eval.py

```diff
@@ -46,51 +47,57 @@ def main(_):
   config.override(FLAGS.hparams)
   config.batch_size = FLAGS.batch_size
   config.val_json_file = FLAGS.val_json_file
-
-  # dataset
-  ds = dataloader.InputReader(
-      FLAGS.val_file_pattern,
-      is_training=False,
-      use_fake_data=False,
-      max_instances_per_image=config.max_instances_per_image)(
-          config)
+  config.nms_configs.max_nms_inputs = anchors.MAX_DETECTION_POINTS
+  base_height, base_width = utils.parse_image_size(config['image_size'])
 
   # Network
   model = efficientdet_keras.EfficientDetNet(config=config)
-  model.build((config.batch_size, None, None, 3))
+  model.build((config.batch_size, base_height, base_width, 3))
   model.load_weights(tf.train.latest_checkpoint(FLAGS.model_dir))
 
-  evaluator = coco_metric.EvaluationMetric(filename=config.val_json_file)
+  augmentations = [] 
```

Comment:
Add a comment:  # in format (height, width, flip)

Reply:
Done.